lib/serialize.js

```diff
@@ -115,15 +116,14 @@ module.exports = function (app, defaults) {
       app: app,
       model: model,
       method: ctx.method.name,
+      meta: ctx.meta ? utils.clone(ctx.meta) : null,
       primaryKeyField: primaryKeyField,
       requestedIncludes: requestedIncludes,
-      host: utils.hostFromContext(ctx),
-      topLevelLinks: {
-        self: utils.urlFromContext(ctx)
-      },
+      host: defaults.host || utils.hostFromContext(ctx),
```

Comment:
`README.md` should have the documentation updated to show the additional supported configuration option.